src/main/scala/org/zalando/zhewbacca/TokenInfo.scala

```diff
@@ -3,13 +3,14 @@ package org.zalando.zhewbacca
 import play.api.libs.functional.syntax._
 import play.api.libs.json.{JsPath, Reads}
 
-case class TokenInfo(accessToken: String, scope: Scope, tokenType: String, userUid: String, clientId: Option[String] = None)
+case class TokenInfo(accessToken: String, scope: Scope, tokenType: String, userUid: String, realm: String, clientId: Option[String] = None)
```

Comment:
Is adding a new (non-optional) constructor parameter considered a breaking change? Should I make it optional instead?

Reply:
It will definitely break in many projects. Better put `realm` as optional after `clientId`